onnxruntime/test/onnx/main.cc

```diff
@@ -312,6 +314,18 @@ int real_main(int argc, char* argv[]) {
       {"scatter_without_axis", "opset 9 not supported yet"},
       {"scan_sum", "opset 9 not supported yet"},
       {"shrink", "opset 9 not supported yet"},
```

Comment:
Cannot exclude `shrink` here because the runtime runs tests against multiple onnx versions (line 42 in tools/ci_build/github/linux/docker/scripts/install_deps.sh). Even though updating to commit id 873ddbbc33c6e54d90c5628387edd391fb651dfc means we have the bug fix, still tests are still run against the original 1.4.1 release commit (till commit id 9e55ace55aad1ada27516038dfbdc66a8a0763db), the test would fail since the bug was originally present.